aostypes/aostypes.go

```diff
@@ -60,20 +60,20 @@ type Time struct {
 	time.Time
 }
 
-// AlertRule describes alert rule.
-type AlertRule struct {
+// AlertRuleParam describes alert rule.
+type AlertRuleParam struct {
 	MinTimeout   Duration `json:"minTimeout"`
 	MinThreshold uint64   `json:"minThreshold"`
 	MaxThreshold uint64   `json:"maxThreshold"`
 }
 
-// ServiceAlertRules define service monitoring alerts rules.
-type ServiceAlertRules struct {
-	RAM        *AlertRule `json:"ram,omitempty"`
-	CPU        *AlertRule `json:"cpu,omitempty"`
-	UsedDisk   *AlertRule `json:"usedDisk,omitempty"`
-	InTraffic  *AlertRule `json:"inTraffic,omitempty"`
-	OutTraffic *AlertRule `json:"outTraffic,omitempty"`
+// AlertRules define service monitoring alerts rules.
+type AlertRules struct {
+	RAM        *AlertRuleParam `json:"ram,omitempty"`
+	CPU        *AlertRuleParam `json:"cpu,omitempty"`
+	UsedDisk   *AlertRuleParam `json:"usedDisk,omitempty"`
```

Comment:
We have agreed to introduce new type for used disk alert. Haven't we? Because with current one we should apply one rule for all partitions.

Reply:
I did but I messed up with comments